net/eoip/patches/100-add_CR_to_pidfile.patch

```diff
@@ -0,0 +1,11 @@
+--- a/eoip.c.orig	2013-09-25 22:50:48.272763057 +0200
++++ b/eoip.c	2013-09-25 22:51:03.804762840 +0200
+@@ -430,7 +430,7 @@
+ 	exit(1);
+ 
+     mfd = fopen(pidfile, "w");
+-    fprintf(mfd,"%d", getpid());
++    fprintf(mfd,"%d\n", getpid());
```

Comment:
```\n``` is not a CR, and why do you need this change?

Reply:
I dont know,but this change on main repository used https://github.com/agustim/openwrt-linux-eoip/commit/1319744c84ea7ccc7ed46a1a8203cc4fe65c22d4

Reply:
> `\n` is not a CR, and why do you need this change?

you think you need to remove the patch?

Reply:
To be honest, I can't judge.
I'm not using eoip so I'm not sure it will work without the patch, and eoip's official repositories do have this change, but there's no detailed explanation...
https://code.google.com/archive/p/linux-eoip/source/default/commits

BTW, I recommend changing ```CR``` to ```LF``` in the patch name.

Reply:
done